src/Objs/defaultPageEditingConfig.ts

```diff
@@ -14,13 +14,21 @@ export const defaultPageEditingConfigAttributes = {
     title: 'Should this page be indexed?',
     description: 'If not, search engines will ignore this page. Default: Yes',
   },
+  layoutShowFooter: {
+    title: 'Show layout footer widgets?',
```

Comment:
* [ ] FYI not sure if an editor will or has to understand what a "layout footer", "footer widget" or a "layout widget" is. Naively I'd go with "Show footer?", but if the PR series is about multiple footer elements, how about "Show footer elements?" or "Show footers?".